app/controllers/placements/support/providers_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Placements::Support::ProvidersController < Placements::Support::ApplicationController
+  def new
+    @provider = Provider.new
+  end
+
+  def create
+    @provider = Provider.find_or_initialize_by(provider_params)
+    if @provider.save
+      redirect_to placements_support_organisations_path
+    else
+      render :new
+    end
+  end
+
+  def check
+    @provider = Provider.new(provider_code: params[:accredited_provider_id])
+    if @provider.valid?
+      accredited_providers = AccreditedProviderApi.call
```

Comment:
I assume Publish doesn't have an endpoint to get only one provider by an id? I'm wondering if we need to get all of them again